bundles/org.openhab.binding.mikrotik/README.md

```diff
@@ -0,0 +1,374 @@
+# Mikrotik RouterOS Binding
+
+This binding integrates [Mikrotik](https://mikrotik.com/) [RouterOS](https://help.mikrotik.com/docs/display/ROS/RouterOS)
+[devices](https://mikrotik.com/products) allowing monitoring of system resources, network interfaces and WiFi clients.
+
+## Supported Things
+
+* `routeros` - An instance of the RouterOS device connection
+* `interface` - A network interface inside RouterOS device
+* `wifiRegistration` - Any wireless client connected to a RouterOS wireless network (regular or CAPsMAN-managed)
+
+
+## Discovery
+
+Discovery is currently not supported, but may be implemented in future versions.
+
+
+## Binding Configuration
+
+The binding has no configuration options, all configuration is done at the Bridge and Thing levels.
+
+
+## Bridge Configuration
+
+To use this binding you need at least one RouterOS-powered device (Bridge) accessible to the host running
+OpenHAB via network.
+Make sure your RouterOS have API enabled by visiting [<kbd>IP -> Services</kbd>](https://wiki.mikrotik.com/wiki/Manual:IP/Services) 
+configuration section in 
+[WinBox](https://wiki.mikrotik.com/wiki/Manual:Winbox). Note the API port number as you'll need it below.
+[SSL API connection](https://wiki.mikrotik.com/wiki/Manual:API-SSL) is not yet supported by this binding.
+To connect to RouterOS API you need to provide user credantials to the bridge config. You may use your current 
+credentials you use to manage device, but it is highly recommended to **create a read-only RouterOS user** since
+this binding only reads data from the device. To do so, proceed to <kbd>System -> Users</kbd> configuration section
+and add a user to the `read` group.
```

Comment:
```suggestion
To connect to the RouterOS API, you will need to provide user credentials for the bridge thing. 
You may use your current credentials that you use to manage your devices, but it is highly recommended to **create a read-only RouterOS user** since this binding only need to read data from the device.
To do this, proceed to <kbd>System -> Users</kbd> configuration section and add a user to the `read` group.
```

New line for each sentence.